workflows/rnadeseq.nf

```diff
@@ -89,16 +88,17 @@ workflow RNADESEQ {
         ch_counts_path,
         ch_metadata_file,
         ch_model_file,
+        ch_gtf,
+
         ch_contrast_matrix,
         ch_contrast_list,
         ch_contrast_pairs,
-        ch_relevel,
         ch_genes,
-        ch_gtf,
+        ch_relevel,
+
         ch_proj_summary_file,
         ch_softwareversions_file,
-        ch_multiqc_file,
-        ch_quote_file
+        ch_multiqc_file
     )
 
     //TODO: Enable this:
```

Comment:
Shouldn't `TODO` strings be removed?

Reply:
I don't know, there are also some that were already present when I started working on deseq...I am keeping some of them to keep track of what changes are necessary in future PRs